morden/sass/_extra-child-theme.scss

```
@@ -25,7 +25,6 @@ a {
 
 	@include media(desktop) {
 		box-shadow: 0 0 30px rgba(#000, 0.2);
```

Let’s bring back the `$content-width-xxl` variable and recompile the styles.

```suggestion
		box-shadow: 0 0 30px rgba(#000, 0.2);
		max-width: $content-width-xxl;
```

Additionally, if you add the following under the Blocks section of `morden/sass/_extra-child-theme.scss` (around line 250), it should fix the margins for gallery and all blocks that need this same floating behavior.

```
/* 
 * Blocks
 */

// Constrain left-aligned blocks
@media only screen and (min-width: 1280px) {
	.entry-content > .wp-block-button:not(.alignleft):not(.alignright),
	.entry-content > .alignleft {
		margin-left: calc( 0.5 * (100% - calc( 782px - 32px)));
		max-width: calc( 0.5 * calc( 782px - 32px));
	}
}

// Constrain right-aligned blocks
@media only screen and (min-width: 1280px) {
	.entry-content > .alignright {
		margin-right: calc( 0.5 * (100% - calc( 782px - 32px)));
		max-width: calc( 0.5 * calc( 782px - 32px));
	}
}
```